REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.5
+julia 0.6.0-pre.alpha.0
```

Comment:
Using `julia 0.6` results in a warning on every `Pkg.update()`:

`WARNING: julia is fixed at 0.6.0-pre.alpha.0 conflicting with requirement for XGBoost: [0.6.0,∞)`

I'll check again when RC1 releases, which should be soon.